CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Changelog
```

Comment:
A good start. I find it nice to see an up-to-date changelog in an open-source package.